src/donate.js

```diff
@@ -222,23 +300,20 @@ window.WebMonetizationScripts.donate = function ({
     // https://example.com/.well-known/pay)
     const spspReceiver = window.WebMonetizationScripts
       .paymentPointerToUrl(paymentPointer)
-
```

Comment:
Can we keep these blank lines that you've removed? I think having some empty lines improves readability and makes the text easier to navigate by paragraph